artview/components/level.py

```diff
@@ -0,0 +1,218 @@
+"""
+level.py
+
+Class instance used for modifying level via Display window.
+"""
```

Comment:
There is also wording [here](https://github.com/gamaanderson/artview/blob/grid/artview/components/level.py#L40)